element-template-generator/core/src/main/java/io/camunda/connector/generator/dsl/CommonProperties.java

```diff
@@ -181,7 +181,7 @@ public static PropertyBuilder deduplicationId() {
                 .pattern(
                     new Pattern(
                         "^[a-zA-Z0-9_-]+$",
-                        "Only alphanumeric characters, dashes, and underscores are allowed"))
+                        "can only contain alphanumeric characters, dashes, and underscores"))
```

Comment:
@chillleader in the PR description, may you please explain why these element template changes are added?

I think the change makes sense - I just want to make this PR easier to read for future reviewers 😄